dist/bootstrap-toc.js

```diff
@@ -48,8 +49,10 @@
         anchor = anchor.replace(/[œ]/g, 'oe');
         anchor = anchor.replace(/[']/g, '');
         anchor = anchor.replace(/[’]/g, '');
-        anchor = anchor.replace(/[^A-Za-z0-9]+/g, '-');
+        anchor = anchor.replace(/\ba\b|\ban\b|\bas\b|\bat\b|\bbefore\b|\bbut\b|\bby\b|\bfor\b|\bfrom\b|\bis\b|\bin\b|\binto\b|\blike\b|\bof\b|\boff\b|\bon\b|\bonto\b|\bper\b|\bsince\b|\bthan\b|\bthe\b|\bthis\b|\bthat\b|\bto\b|\bup\b|\bvia\b|\bwith\b/g, '');
```

Comment:
Pathauto dynamically generates a regex to strip certain words from the path. This is a copy of the current settings. If the settings in pathauto change we have to change this.

We could consider sending the regex string as a js var that this file could read instead.

See pathauto.inc around like 200 or so.